oauth-postman.md

```diff
@@ -1,50 +1,60 @@
-|Token Name|Enter any name you would like|
-|Grant Type|Authorization Code|
-|Callback URL|Your OAuth client redirect URL|
-|Auth URL|https://ims-na1.adobelogin.com/ims/authorize/v1|
-|Access Token URL|https://ims-na1.adobelogin.com/ims/token/v1|
-|Client ID|Enter your client id here|
-|Client Secret|Enter your client secret here|
-|Scope|openid,AdobeID,read_organizations,additional_info.projectedProductContext,additional_info.job_function|
-|State|Can be left empty|
-|Client Authentication|Send client credentials in body|
 
 
+4.  On the **GET NEW ACCESS TOKEN** form, provide the following values:
+
+| Parameter                 | Value                                                                                                          |
+| ------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| Token Name            | Specify a name                                                                                                 |
+| Grant Type            | Select **Authorization Code** from the dropdown                                                                               |
+| Callback URL          | Enter your OAuth client default redirect URI                                                                   |
+| Auth URL              | <https://ims-na1.adobelogin.com/ims/authorize/v1>                                                              |
+| Access Token URL      | <https://ims-na1.adobelogin.com/ims/token/v1>                                                                  |
+| Client ID             | Enter your Client ID from Adobe I/O                                                                                          |
+| Client Secret         | Enter your Client Secret from Adobe I/O                                                                                        |
+| Scope                 | Enter `openid AdobeID read_organizations additional_info.projectedProductContext additional_info.job_function` |
+| Client Authentication | Select **Send client credentials in body** from the dropdown                                                                     |
+
 
 ![Postman Oauth Step 3](/images/postman-oauth2-step3.png)
 
-Click `Request Token`. You will be prompted to login and then you will get your access token on the next screen
+5.  Click **Request Token**. You will be prompted to login. Click the **Use Token** button at the bottom of the resulting pop up screen. The token automatically populates the **Available Token** field.
+
+6. Enter the request URI path you want to call, including your Global Company ID as shown below, and then click the **Preview Request** button. This updates the request header with the token value.
+
+
+  ![Postman Oauth Step 4](/images/postman-oauth2-step4.png)
 
+  [!NOTE] If you have already requested other tokens in Postman, you can select them from the **Available Tokens** dropdown.
```

Comment:
The [!NOTE] syntax is not rendering properly

Reply:
Yes. Thanks for catching that. That is from a stylesheet that I found out does not currently apply to Adobe I/O. Can you please share with me how you are previewing this to see how it renders on Adobe I/O? Very helpful for me.

Reply:
I previewed in on Github. There is a button to the left of the `copy path` button when viewing the file changes that will show you the rendered markdown.

Reply:
Oh ok. I thought you had a special preview of it on Adobe I/O, not just github. Thanks.